app/src/features/Batches/BatchList.tsx

```diff
@@ -133,58 +137,67 @@ const BatchList = (): JSX.Element => {
     console.log(batchId);
   };
 
-  if (isBatchesLoading) return <CircularProgress />;
+  if (isBatchesLoading)
+    return (
+      <div className={classes.loader}>
+        <CircularProgress />
+      </div>
+    );
   return (
     <>
       <div className={classes.accordions}>
         {batches?.results &&
           batches.results.map((batch) => (
-            <Accordion key={`batch-${batch.id}`} className={classes.accordion}>
+            <Accordion className={classes.accordion} key={`batch-${batch.id}`}>
```

Comment:
we won't be displaying errors directly.. maybe we can use something else than accordions.

Reply:
Where will we display the errors ? In a modal or another page ?